Makefile

```diff
@@ -118,8 +118,14 @@ test-server: install-server ## Run unit tests
 test-ui: lint-ui ## Run frontend tests
 	npm test
 
-test-ui-debug: lint-ui
-	npm run test-debug
+test-ui-integration: ## Run frontend cypress integration tests
+	npm run test:integration
+
+test-ui-unit: ## Run frontend jest unit tests
+	npm run test:unit
+
+test-ui-debug: ## Open cypress integration test debugger
+	npm run test:integration:debug
 
 test: test-server test-ui ## Run all tests
 
```

Comment:
Something seems odd here, although we are adding some new targets to make, they are not part of the `npm test`?

Reply:
And they aren't a dependency of `test-ui`.  Seems like `test-ui` should depend on `lint-ui test-ui-unit test-ui-integration` and then not call `npm test` since those two dependencies should perform that equivalent.

Reply:
`make test-ui` runs `npm test` which calls both `npm run test:unit` and `npm run test:integration` and is generally the test command we want to run. But in an earlier review it was requested to add `make` targets for running the unit and integration tests separately. So `make test-ui-integration` and `make test-ui-unit` we added that call those nom scripts directly. Does that help clear things up? We could change this to have `make test-ui` just call the `test-ui-integration` and `test-ui-unit` make targets instead, but that would just be doing the same thing as it does now, just using make rather than npm to handle combining the calls.

Reply:
Yes that makes sense.  However, I think we should perform the same tests we do as part of CI that we do as part of our dev workflow.  As a result, I suggested that `test-ui` be the primary target for ui-testing and it seems like it should call the two new targets.  The end result is the same as if `npm test` was run anyway - correct?

Reply:
that is correct, if you would rather have the make file call each than the npm script call each there no difference in actually execution.

Reply:
ok.  Someone wanted the extra targets - which make it easier for folks to run these - so I agree with that.  My only ask was [stated above](https://github.com/elyra-ai/elyra/pull/818#issuecomment-673140626).  So if there _are_ new targets that call the `npm` equivalents, then there would be no need to have `test-ui` itself call `npm` also.

Reply:
Just changed `make test-ui` to depend on `lint-ui test-ui-unit test-ui-integration`.